test/run.py

```diff
@@ -3,13 +3,15 @@
 
 def run_tests():
 
-    import tests.api.v1_1 as api_1_1 
+    import tests.api.v1_1 as api_1_1
     import tests.api.v2_0 as api_2_0
     import tests.api.redfish_1_0 as api_redfish_1_0
+    import tests.benchmark as benchmark
```

Comment:
This means benchmark test will always run at the end of Jenkins test. Does it by design ?